src/binary-reader.cc

```diff
@@ -2182,7 +2182,7 @@ Result BinaryReader::ReadTypeSection(Offset section_size) {
       uint8_t type;
       CHECK_RESULT(ReadU8(&type, "type form"));
       ERROR_UNLESS(type == 0x60, "unexpected type form (got " PRItypecode ")",
-                   WABT_PRINTF_TYPE_CODE(type));
+                   "", static_cast<int32_t>(type));
```

Comment:
I think you can just replace `PRItypecode` here is `%d` and drop the cast.

Reply:
Thanks for the review. I updated it following your comment.